content/components/relative-time.mdx

```diff
@@ -1,16 +1,24 @@
 ---
-title: Relative time
+title: Dates & Times
 status: Experimental
 description: A relative time displays the time difference between the current time and the specified date-time.
 ---
 
 import {Box} from '@primer/react'
 
-## Content
+## Overview
+
+All dates and times displayed should use the Relaive Time component, which is able to format and localise dates into the users given locale, as well display times relative to the current date, while also updating in real time.
```

Comment:
suggestions according to this ADR on component naming: https://github.com/github/primer/blob/main/adrs/2022-02-15-spelling-of-component-names.md

```suggestion
All dates and times displayed should use the relative time component, which is able to format and localise dates into the users given locale, as well as display times relative to the current date, while also updating in real-time.
```